acceptance-tests/features/submit_case.feature

```diff
@@ -1,4 +1,4 @@
-@US07
+@submit_case
```

Comment:
Why did we take out the user story number, we should maybe include both e.g. ```@submit_case_US07```, submit_case doesn't mean much to me but maybe you had a good reason to do this?